Refactoring ahead of multi-plot metrics docs (#1996)
wandb/docs: models/runs.mdx

2. Select the **Workspace** tab to view your project's workspace.
3. Search and select the name of the run that contains an infrastructure issue. If CoreWeave detected an infrastructure issue, one or more red vertical lines with an exclamation mark overlay the run's plots. 
4. Select an issue on a plot or select the **Issues** button in the top right of the page. A drawer appears that lists each issue reported by CoreWeave. 

<Tip>
To views runs with infrastructure issues at a glance, pin the **Issues** column to your W&B Workspace to view runs that logged an issue at a glance. For more information about how to pin a column, see [Customize how runs are displayed](/models/runs/#customize-how-runs-are-displayed).
</Tip>

The **Overall Grafana view** at the top of the drawer redirects you to the SLURM job's Grafana dashboard, which contains system-level details about the run. The **Issues summary** describes the root error that the SLURM job reported to CoreWeave Mission Control. The summary section also describes any attempts to automatically resolve the error made by CoreWeave.

<Frame>
    <img src="/images/app_ui/cw_wb_observability.png"  />
</Frame>

> The **All Issues** list all issues that occurs during the run in chronological order, with the most recent issue at the top. The list contains the job issue and node issue alerts. Within each issue alert is the name of the issue, the timestamp when the issue occurred, a link to the Grafana dashboard for that issue, and a brief summary that describes the issue.

Reviewer: ```suggestion
**All Issues** lists all issues that occur during the run in chronological order, with the most recent issue at the top. The list contains the job issue and node issue alerts. Each issue alert includes the issue name, the timestamp of the issue, a link to the Grafana dashboard for that issue, and a brief summary.
```

| Node Availability & Readiness | `KubeNodeNotReadyHGX`, `NodeExtendedDownTime` |
| GPU/Accelerator Errors | `GPUFallenOffBusHGX`, `GPUFaultHGX`, `NodeTooFewGPUs` |
| Hardware Errors | `HardwareErrorFatal`, `NodeRAIDMemberDegraded` |
| Networking & DNS | `NodeDNSFailureHGX`, `NodeEthFlappingLegacyNonGPU` |
| Power, Cooling, and Management | `NodeCPUHZThrottle`, `RedfishDown` |
| DPU & NVSwitch | `DPUNcoreVersionBelowDesired`, `NVSwitchFaultHGX` |
| Miscellaneous | `NodePCISpeedRootGBT`, `NodePCIWidthRootSMC` |

For detailed information on error types, see the [SLURM Job Metrics on the CoreWeave Docs](https://docs.coreweave.com/docs/observability/managed-grafana/sunk/slurm-job-metrics#job-info-alerts#job-info-alerts).

### Debug infrastructure issues

Each run that you create in W&B corresponds to a single SLURM job in CoreWeave. You can view a failed job's [Grafana](https://grafana.com/) dashboard or discover more information about a single node. The link within the **Overview** section of the **Issues** drawer links to the SLURM job Grafana dashboard. Expand the **All Issues** dropdown to view both job and node issues and their respective Grafana dashboards. 

<Note>

> **Note**

Reviewer: ```suggestion
```
